ONSdigital/eq-author-app
# Remove custom keyword & replace with error message

### What is the context of this PR?

Tidy up the way error messages are created. Removes the use of custom keywords and uses errorMessage instead

### How to review 

1. Create questionnaire
2. Add answer with routing and/or skip logic
3. Check network tab for validationerror info for corresponding expressions
